docs/01-start-here/01-installation-steps.md

```diff
@@ -56,7 +58,7 @@ Containers can be thrown away very easily. To do so:
 ## Local machine
 You need a Mac or Linux PC (Ubuntu).
 
-Before checking out the repository you may need to [add an SSH key to your GitHub account](https://help.github.com/articles/generating-ssh-keys/). Before pushing changes you may need to [create an access token](https://help.github.com/articles/creating-an-access-token-for-command-line-use/).
```

Comment:
I've shifted that up to the start of the enclosing section 'Local Test Server setup', as I think it should probably apply to the Docker instructions too?